WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -53,8 +48,22 @@ struct AccountSettingsService {
         }
     }
 
+    /// Emits a value when the settings for the associated account change.
+    var settings: Observable<AccountSettings?> {
+        let notificationCenter = NSNotificationCenter.defaultCenter()
+        let notificationObserver = notificationCenter.rx_notification(NSManagedObjectContextDidSaveNotification, object: context)
+        // This was the simplest implementation. If performance is an issue, we could try
+        // adding `distinctUntilChanged` or `filter` on the notification userInfo and only
+        // emit if the changed objects include the observed account.
+        return notificationObserver.map(getSettings).startWith(getSettings())
```

Comment:
Every time the associated context saves, but yes. The observer is removed when the subscription to the observable is disposed, which in this case it's [when the controller deallocates](https://github.com/wordpress-mobile/WordPress-iOS/pull/4691/files#diff-b9a1b30bff0cce9607a4aa0468d69806R19). As I mentioned with the comment on memory management, I've seen the controller/VC not being released properly. This isn't that much of a problem, since in the next PR, the controller unsubscribes when the view disappears.